src/solver/variable/state.cpp

```diff
@@ -55,6 +55,9 @@ void ThermalState::StateForAnArea::initializeFromArea(const Data::Area& area)
     numberOfUnitsONbyCluster.resize(count);
     thermalClustersOperatingCost.resize(count);
     PMinOfClusters.resize(count);
+    unitCountLastHour.resize(count);
+    productionLastHour.resize(count);
```

Comment:
**productionLastHour** : this variable is never initialized, it used to be though.

Further more, it seems like it is used only in `State::initFromThermalClusterIndex`.
Could we not make it a local variable and remove it as a data member of class `ThermalState::StateForAnArea` ?

Reply:
And I have the same remark  for **unitCountLastHour**.
**pminOfAGroup** is not initialized but is used in 2 different `State` methods, so I guess it should remain a `ThermalState::StateForAnArea` data member.